entity-service/src/test/kotlin/com/egm/stellio/entity/service/EntitiesEventServiceTests.kt

```diff
@@ -0,0 +1,115 @@
+package com.egm.stellio.entity.service
+
+import com.egm.stellio.shared.model.AttributeReplaceEvent
+import com.egm.stellio.shared.model.EntityCreateEvent
+import com.egm.stellio.shared.model.EntityDeleteEvent
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_CORE_CONTEXT
+import com.egm.stellio.shared.util.loadSampleData
+import com.egm.stellio.shared.util.matchContent
+import com.egm.stellio.shared.util.toUri
+import com.ninjasquad.springmockk.MockkBean
+import io.mockk.every
+import io.mockk.slot
+import io.mockk.verify
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.cloud.stream.binding.BinderAwareChannelResolver
+import org.springframework.messaging.Message
+import org.springframework.test.context.ActiveProfiles
+
+@SpringBootTest(classes = [EntitiesEventService::class])
+@ActiveProfiles("test")
+class EntitiesEventServiceTests {
```

Comment:
Here you should only test the message is sent to the expected topic (one test should be enough). Serialization of events should be covered in shared module (where events are defined)